### PR TITLE
Added descriptions to OS.get_splash_tick_msec() and OS.get_window_safe_area()

### PR DESCRIPTION
**Pull request by Anutrix**
I think it's correct but do check the grammar.

### CODE REVIEW

**Comment by Anutrix** [comment]
My bad. Just realized  after seeing Doc Sprint page that Akein seems to working on OS docs. Should I close this PR?

**Comment by akien-mga** [comment]
No it's fine, I haven't started proofreading OS yet.

File: doc/classes/OS.xml [hunk 1]
```diff
@@ -360,6 +360,7 @@
 			<return type="int">
 			</return>
 			<description>
+				Returns the amount of time in milliseconds it took for the boot logo to appear.
```
Comment (akien-mga) [inline doc/classes/OS.xml]: TIL about this method :O
I don't really see why this information is relevant enough to expose in the first place, but the docs are good :)

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

File: doc/classes/OS.xml (hunk 1 above)
Comment (Anutrix) [reply doc/classes/OS.xml]: Understood.

Comment (akien-mga) [reply doc/classes/OS.xml]: To be clear, I did not mean this for you, this is just me wondering why we have this method in the first place.

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.2.

**Pull request merged**